configs/xlml/jax/solutionsTeam_pax_latest_supported_config.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utilities to construct configs for solutionsTeam_pax_latest_supported DAG."""
+
+import uuid
+from apis import gcp_config, metric_config, task, test_config
+from configs import test_owner, vm_resource
+from configs.xlml.jax import common
+
+
+def get_pax_resnet_config(
```

Comment:
@RissyRan is `get_pax_resnet_config` the best name for this function? How does it relate to resnet?

Reply:
Good catch! I forget to update the helper function name when use the template from JAX/TF. Updated to `get_pax_lm_config` to get language model configs in pax.